src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/repository/UserPersonDetailRepositoryTest.kt

```diff
@@ -791,6 +791,7 @@ class UserPersonDetailRepositoryTest {
           "IBRAGIM.MIHAIL",
           "MARIAN.CHESED",
           "LEOPOLDO.CHESED",
+          "LEOPOLDO.CHESED",
```

Comment:
doesn't this mean we have a duplicate again?

Reply:
yes, it uses the existing criteria hence reverted the test. Will create a ticket to address duplicates